gui/packages/ubuntupro/lib/pages/landscape/landscape_model.dart

```diff
@@ -294,22 +216,20 @@ class LandscapeSelfHostedConfig extends LandscapeConfig {
 
   @override
   bool get isComplete =>
-      !fqdnError && fileError == FileError.none && fqdn.isNotEmpty;
+      !fqdnError && fqdn.isNotEmpty && fileError == FileError.none;
```

Comment:
Ha, this indeed looks odd. I believe this is because I actually constructed the "new" manual configuration from the SaaS objects, and moved the previous "custom" pieces into it. The previous custom config had a file consideration, but SaaS did not and I added it, thus this weird diff